std_msgs/Header header

int32 drone_id

IslandBox[] islands